packages/frontapp/src/app/components/UserList/index.tsx

```diff
@@ -36,17 +40,23 @@ const Description = styled.p`
 
 const UserList = () => {
   const { data, loading } = useGetAllPersonsQuery();
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    if (data) dispatch(setSearchReaders(data?.getAllPersons));
+  }, [data]);
+  const { searchReaders } = useAppSelector((state) => state.readers);
```

Comment:
it is better to take the state from the redax above before useEffect, so the code looks more beautiful and readable